frontend/src/components/DatePicker.js

```diff
@@ -61,6 +61,8 @@ const DateInput = ({
                 onDateChange={(d) => {
                   const newDate = d ? d.format(dateFmt) : d;
                   onChange(newDate);
+                  const input = document.getElementById(name);
+                  if (input) input.focus();
```

Comment:
Ideally, we would want to use `ref `to switch focus, however, I run into problems with that approach.